.babelrc

```diff
@@ -1,10 +1,5 @@
 {
-  "presets": [
-    ["@babel/preset-env", {
-      "modules": false,
-      "targets": {
-        "browsers": ["> 1%", "last 2 versions", "not ie <= 8"]
-      }
-    }]
+  "plugins": [
+    "transform-es2015-modules-commonjs"
   ]
-}
```

Comment:
There's a random change here that has to do with a newline character being removed from the end of the file. We should probably decide as a team whether we should enforce newline at the end of a file or not so that we don't have this extraneous change in commits every time someone with a different newline configuration pushes a change to a particular file.